add entry for 1.0.0 release to fix version in pkg-config file

--- CHANGELOG
+++ CHANGELOG
@@ -3,6 +3,9 @@
     The AV1 encoder's default keyframe interval changed to 128 from 9999.
     Support for armv6 was removed.

+2019-03-16 v1.0.0 "AOMedia Codec 1"
+  Current release
+
 2016-04-07 v0.1.0 "AOMedia Codec 1"
   This release is the first Alliance for Open Media codec.
 2015-11-09 v1.5.0 "Javan Whistling Duck"
